src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/ReturnCountCheck.java

```diff
@@ -137,6 +230,51 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * public class MyClass {
+ *   public class FirstTest {
+ *     public FirstTest() {
```

Comment:
do not create inner classes
use:
```
public class Test {
    public Test() {
    } // OK

    public Test(int i) {
      return;
    } // violation is expected but not placed until #11212
```

Please also update all lines to have `violation` or `ok` comment on line where violation is placed if it exists - method signature.

for line where we have defect, please place comment "violation is expected but not placed until #11212 "

Reply:
Done